src/edu/stanford/dlss/was/WasapiFile.java

```diff
@@ -0,0 +1,128 @@
+package edu.stanford.dlss.was;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Class corresponding to JSON returned by WASAPI that represents WebdataFile
+ * @see https://github.com/WASAPI-Community/data-transfer-apis/blob/master/ait-implementation/wasapi/implemented-swagger.yaml#L132-L183
+ */
+public class WasapiFile {
+
+  private String filename;
+  @JsonProperty("checksum")
```

Comment:
Would it be possible to group the regular class members and the annotated members such that they don't interleave? This would seem to make it easier to read?